streams/src/test/java/org/apache/kafka/streams/integration/QueryableStateIntegrationTest.java

```diff
@@ -265,7 +265,7 @@ private void verifyAllKVKeys(final StreamRunnable[] streamRunnables, final Kafka
                 public boolean conditionMet() {
                     try {
                         final StreamsMetadata metadata = streams.metadataForKey(storeName, key, new StringSerializer());
-                        if (metadata == null) {
+                        if (metadata == null || metadata.equals(StreamsMetadata.NOT_AVAILABLE)) {
```

Comment:
They have a different meaning. `null` means that the `storeName` doesn't exist - perhaps it should throw an exception in this case. 
`StreamsMetadata.NOT_AVAILABLE` means the streams haven't (re-)initialized